api2-war/src/main/java/eu/europeana/api2/v2/service/translate/MetadataTranslationService.java

```diff
@@ -292,11 +287,20 @@ private void getSearchValueToTranslateForField(BriefBean bean, Field field, Stri
      * @param translatedValues list of translated values to be added
      * @param targetLanguage language for the translated values
      */
-    private void addTranslationToObject(Object object, String fieldName, List<String> translatedValues, String targetLanguage) {
+    private void addTranslationToObject(FullBean bean, Object object, String fieldName, List<String> translatedValues, String targetLanguage) {
```

Comment:
We should document the new Fullbean parameter. What is it used for?

Reply:
i have documented it